encoding/protobuf/error.go

```diff
@@ -25,12 +25,12 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/gogo/googleapis/google/rpc"
-	"github.com/gogo/protobuf/proto"
-	"github.com/gogo/status"
+	"github.com/golang/protobuf/proto"
 	"go.uber.org/yarpc/api/transport"
 	"go.uber.org/yarpc/internal/grpcerrorcodes"
 	"go.uber.org/yarpc/yarpcerrors"
+	spb "google.golang.org/genproto/googleapis/rpc/status"
```

Comment:
nit: let's rename this statuspb, even if spb is kind of the same it will help for readability of the code